content/news/052/index.md

```diff
@@ -39,7 +39,7 @@ Feel free to send PRs about your own projects!
 - [Blog Posts](#blog-posts)
 - [Jobs](#jobs)
 - [Misc. Links](#misc-links)
-- [Future News](#future-news)
+- [Future news](#future-news)
```

Comment:
"Blog Posts" but "Future news"? Are you sure?

Reply:
Oops, this was done automatically by some markdown plugin I think. It's probably because the section name is lower case, I'll fix that instead 👍🏻